packages/editor-ui/src/components/Node/NodeCreator/viewsData.ts

```diff
@@ -124,7 +125,9 @@ export function AIView(_nodes: SimplifiedNodeType[]): NodeView {
 		value: AI_NODE_CREATOR_VIEW,
 		title: i18n.baseText('nodeCreator.aiPanel.aiNodes'),
 		subtitle: i18n.baseText('nodeCreator.aiPanel.selectAiNode'),
-		info: i18n.baseText('nodeCreator.aiPanel.infoBox'),
+		info: i18n.baseText('nodeCreator.aiPanel.infoBox', {
+			interpolate: { link: WEBSITE_TEMPLATES_URL },
```

Comment:
maybe we can point to AI category https://n8n.io/workflows/?categories=25 instead? instead of the collection?

Reply:
Good point, I think it makes sense.